modules/build-macros/src/main/scala/scala/build/Ops.scala

```diff
@@ -187,4 +188,18 @@ object Ops {
           Left(errors0)
       }
   }
+
+  extension [T](items: Seq[T]) {
```

Comment:
Can you add comment explaining why it was necessary to implement our own `distinctBy` function? Additionally, I think we should consider renaming it to avoid potential conflicts with the `distinctBy` function from the standard library when importing `Ops.scala`.

Reply:
I mean, afaik there is no standard-lib equivalent of `distinctBy`... is it worth putting in a comment as an explanation?

Reply:
technically we do have tests verifying that already as a side effect, but I can add smth dedicated

Reply:
> I mean, afaik there is no standard-lib equivalent of distinctBy
Right, so providing some explanation should be sufficient

Reply:
Should be fixed, added some tests and a scaladoc for `distinctBy`